test/powershell/Modules/PSDesiredStateConfiguration/MOF-Compilation.Tests.ps1

```diff
@@ -7,7 +7,14 @@ Describe "DSC MOF Compilation" -tags "CI" {
     }
 
     BeforeAll {
-        $SkipAdditionalPlatforms = (Get-PlatformInfo) -match "alpine|raspbian"
+        $platformInfo = Get-PlatformInfo
+        $SkipAdditionalPlatforms =
+            ($platformInfo.Platform -eq "alpine") -or
+            ($platformInfo.Platform -eq "raspbian") -or
```

Comment:
Maybe continue to use `-match "alpine|raspbian"` here?

Reply:
Fixed